docs/section19/19-4.md

```diff
@@ -0,0 +1,102 @@
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE     
+**Appendix G Section:** Exception to G3.1.2.2.1          
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each Space  
+
+**Applicability Checks:**  
+1. Applies to residential dwelling unit spaces only.  
+  
+   
+**Function Calls:**  
+1. get_component_by_id()  I think this is supposed to return the object associated with an ID?
+2. get_most_used_weekday_hourly_schedule()  To be developed by RCT team. I created a shell for this so they know what we need.  
+
+
+
+## Rule Logic:   
+**Applicability Check 1 - Check if the space is a residential dwelling unit**  
+- For each space in the B_RMI: `for space in B_RMI...Space:`   
+    - Reset bldg_type to "": `bldg_type = ""`  
+    - Reset the building area defined boolean variable to false: `bldg_area_is_defined = false`  
+    - Reset the building_area_is_MF_dormitory_or_hotel boolean variable to false: `building_area_is_MF_dormitory_or_hotel = false`  
+    - Reset space_lighting_space_type_is_defined to true: `space_lighting_space_type_is_defined = true`  
+    - Reset space type is dwelling unit boolean variable to false: `space_lighting_space_type_is_dwelling_unit = false` 
+
+    - Get the zone that the space is part of by looping through zones and checking is the space is associated with the zone: `for zone in B_RMI...Zone:`  
+        - Check if the space is associated with the zone, if it is then set is equal to the applicable_zone: `if space in zone.spaces: applicable_zone = zone`  
+    
+    - Check if the lighting and ventilation space types are both NOT defined: `if space.lighting_space_type == Null and space._ventilation_space_type == Null: ` 
+        - Set space_lighting_space_type_is_defined = false: `space_lighting_space_type_is_defined = false`  
+        - Loop through the building segments to determine which building segment the zone is apart of: `for bldg_seg in B_RMI...Building.building_segments:`  
+            - Check if the zone is in bldg_seg.zones, if yes then set the bldg_type to the building area lighting type associated with the building segment: `if applicable_zone in bldg_seg.zones: bldg_type = bldg_seg.lighting_building_area_type`
+            - Check if the lighting building area type was defined: `if bldg_type != Null: bldg_area_is_defined = true`  
+            - Check if the lighting building area type is multifamily, dormitory, or hotel/motel: `if bldg_type in ["DORMITORY","HOTEL_MOTEL","MULTIFAMILY"]: building_area_is_MF_dormitory_or_hotel = true`  
+    
+    - Check if space_lighting_space_type_is_defined = True, if yes carry on with logic, if not go to rule assertions: `if space_lighting_space_type_is_defined = true:`  
```

Comment:
Should like 44 be check if space lighting space type or space ventilation space type is defined, then carry on with logic?

Reply:
Makes sense, change made.